src/main/java/com/mattmerr/beets/commands/PlayCommand.java

```diff
@@ -71,9 +84,11 @@ public Mono<Void> execute(SlashCommandEvent event) {
         log.error("Error looking up beet by name", e);
       }
     }
-    AudioTrack track = beetLoader.getTrack(beet);
+
+    List<AudioTrack> tracks = beetLoader.getTracks(beet, multiqueue);
     final var beetCapture = beet;
     if (delayMinutes.isPresent()) {
+      AudioTrack track = tracks.iterator().next(); // Cannot multiqueue and delay, so this will always have 1 element.
```

Comment:
If we're keeping GetTrack we should use that

Reply:
Done.